### DISCUSSION TITLE
Remap Hyper

### DISCUSSION THREAD

**Discussion by ghost** [WootingKb/wootility-issues#316] [Q&A]
There is no hyper key in the remap list, is it possible to map this?

**Comment by gitraar** [comment]
You can map a key to be a DKS key where you use the four bindings for shift, alt, control, and meta.

**Comment by ghost** [comment]
So no?

**Comment by gitraar** [comment]
I was just trying to help you map the Hyper functionality to a key without there being a specific button in the configuration to do so. In practice, it would be a Hyper key.

Whether it will be possible to map it any other way in the future, I don't know. I don't work for Wooting.

**Reply by ghost** [reply]
Ctrl Alt Shift Win/Mod4 isn't actually the real Hyper. Can I map it in xorg.conf sure, but then the software has little to no point if it can't map what you need and mod it keyboard side.
